src/auth/auth.controller.ts

```diff
@@ -54,4 +61,90 @@ export class AuthController {
 
     res.sendStatus(HttpStatus.OK);
   }
+
+  @ApiExcludeEndpoint()
+  @Get('login')
+  async jwtLogin(
+    @Query(
+      new ValidationPipe({
+        errorHttpStatusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+        transform: true,
+      }),
+    )
+    { token }: LoginQueryDto,
+    @Res() res: Response,
+  ): Promise<void> {
+    const redirectUrl = `${this.config.get<string>(
```

Comment:
what do you think about redirect to /login ? no strong opinion but it seems a more common to do that for login error.